next/components/Molecules/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,111 @@
+import ChevronLeft from '@assets/images/chevron-left.svg'
+import ChevronRight from '@assets/images/chevron-right.svg'
+import cx from 'classnames'
+import React, { useRef } from 'react'
+
+import { VerticalCardButton } from '../../HomePage/VerticalCardButton'
+
+interface SectionPromosProps {
+  listClassName?: string
+  itemClassName?: string
+  items: { element: JSX.Element | null; key: string | undefined }[]
+  shiftIndex?: number
+  visibleItems?: number
+}
+
+const Carousel = ({
+  listClassName,
+  itemClassName,
+  items,
+  shiftIndex = 1,
+  visibleItems = 3,
+}: SectionPromosProps) => {
+  const [currentItem, setCurrentItem] = React.useState(0)
+
+  const scrollerRef = useRef<HTMLUListElement>(null)
+
+  const scrollToImage = (i: number, instant = false) => {
+    setCurrentItem(i)
+    if (!scrollerRef.current) return
+    const offset = (scrollerRef.current.scrollWidth / items.length) * i
+
+    scrollerRef.current?.scroll({
+      left: offset,
+      behavior: instant ? 'auto' : 'smooth',
+    })
+  }
+
+  const nextImage = () => {
+    const nextIdx = currentItem + shiftIndex
+    const maxIdx = items.length - visibleItems
+    if (nextIdx >= maxIdx + shiftIndex) {
+      // If on last page, go to 0
+      // scrollToImage(0, true)
+      return
+    }
+
+    if (nextIdx > Math.min(maxIdx - shiftIndex, maxIdx - visibleItems)) {
+      scrollToImage(nextIdx)
+      return
+    }
+
+    scrollToImage(nextIdx)
+  }
+
+  const previousImage = () => {
+    scrollToImage(currentItem - shiftIndex)
+  }
+
+  const sliderControl = (isLeft: boolean) =>
+    items.length >= visibleItems ? (
+      <VerticalCardButton
+        onPress={isLeft ? previousImage : nextImage}
+        size="large"
+        className={cx('absolute top-0 bottom-0 z-10 my-auto', {
```

Comment:
About positioning, I don't know where to put this comment petter :D 

According to figma, the button should be on the edge of the first/last card. I think that if it is in the connrect size, it should fit in the screen. We actually do not need to enlarge the button on hover if it causes layout problems.

I also noticed that there is negative margin also on medium size screens where it shouldn't be:
![image](https://user-images.githubusercontent.com/19418224/212718247-86b6f26c-abaa-47f3-8103-7c20f1b9dcca.png)